packages/payload/src/admin/components/views/API/index.tsx

```diff
@@ -204,6 +224,22 @@ export const API: React.FC<EditViewProps> = (props) => {
     fetchData()
   }, [i18n.language, fetchURL, authenticated])
 
+  React.useEffect(() => {
+    let apiActions = []
+
+    if (collection && hasAPIActions(collection.admin.components.views.Edit)) {
```

Comment:
Couldn't this be simplified so that you don't need the `hasAPIActions` function by doing this?
```
apiActions = (collection || global)?.admin?.components?.views?.Edit?.API?.actions
```

Reply:
@DanRibbens I definitely don't need the `hasAPIActions` function but using the above suggested approach gives me type errors saying cannot find `API`

So I think the workaround here is this:
```
let apiActions = []

if (collection && 'API' in collection.admin.components.views.Edit && 'actions' in 
    collection.admin.components.views.Edit.API) {
      apiActions = collection.admin.components.views.Edit.API.actions
} else if (global && 'API' in global.admin.components.views.Edit && 'actions' in 
    global.admin.components.views.Edit.API) {
      apiActions = global.admin.components.views.Edit.API.actions
}
```

Reply:
OK I pulled it down and did this instead:

```
const editConfig = (collection || global).admin.components.views.Edit
apiActions = 'API' in editConfig && 'actions' in editConfig.API ? editConfig.API.actions : []
```
It isn't beautiful but it is an improvement from having if/else and targeting the same properties over and over.